apps/nuxt3-ssr/components/CohortCard.vue

```diff
@@ -85,21 +76,18 @@ const isShowingMobileMoreText = computed(() => {
         {{ cohort?.description }}
       </p>
 
-      <p class="text-body-base mt-5 block xl:hidden">
+      <p v-if="cohort?.description" class="text-body-base mt-5 block xl:hidden">
         {{
-          isShowingMobileMoreText
-            ? cohort?.description
-            : `${cohort?.description?.substring(
-                0,
-                mobileShowMoreTextLength
-              )}...`
+          truncate
+            ? `${cohort?.description?.substring(0, cutoff)}...`
```

Comment:
for now okay, but don't we want to use a component for this that truncates so we can easily do it everywhere?

Reply:
yes, good idea , but still suggest we do this bug fix first